tests/ssl-opt.sh

```diff
@@ -8968,7 +8968,8 @@ run_test    "TLS 1.3: CertificateRequest check - openssl" \
             "$O_NEXT_SRV -msg -tls1_3 -num_tickets 0 -no_resume_ephemeral -no_cache -Verify 10" \
             "$P_CLI debug_level=4 force_version=tls13 " \
             1 \
-            -c "CertificateRequest not supported"
+            -c "=> parse certificate request" \
```

Comment:
Add the check of "got a certificate request" to ensure that we parsed the certificate request successfully?